README.md

```diff
@@ -72,35 +72,24 @@ If there's a typing error in your Python code, then the test will fail. Using `-
 
 ## Installation
 
-**1. Create a file (`tools/typing/mypy_version.txt` in the example below) that will specify the version of `mypy` to use.** You will pass the Bazel label for
-this file to the `deps()` function in `@mypy_integration//repositories:deps.bzl`, which below is named
-`mypy_integration_deps(...)`:
+`mypy_integration` expects the user to provide the `mypy` dependency.
+Given not every mypy version is compatible to all Python versions and mypy's transitive dependencies can differ based on the Python version `mypy_integration` cannot offer a `mypy` which satisfies all potential users.
```

Comment:
```suggestion
Given not every mypy version is compatible to all Python versions and mypy's transitive dependencies can differ based on the Python version, `mypy_integration` cannot offer a `mypy` which satisfies all potential users.
```

Reply:
Will fix this in another PR so this large PR can be merged without a new review.

Reply:
Ah right sorry, at work when I approve a PR the author hits merge when they're ready, so here out of habit I approved and then didn't hit merge. I'm happy to merge whenever you're ready.